ShopifySharp/Entities/SalesChannel/CheckoutSalesChannel.cs

```diff
@@ -0,0 +1,249 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace ShopifySharp.Entities.SalesChannel
+{
+    [JsonObject(ItemNullValueHandling = NullValueHandling.Ignore)]
+    public partial class CheckoutSalesChannel
```

Comment:
There are quite a few instances of `object` in here that should be turned into classes or use existing classes where possible. For example, there are classes for the `DiscountCode` and `Order` objects already.

Reply:
Removed fields for which there is no description in the documentation, for the remaining fields assigned existing types